Ginger/Ginger/Actions/ActionEditPages/ValidationDBPage.xaml.cs

```diff
@@ -246,6 +258,24 @@ private void DBNameComboBox_SelectionChanged(object sender, SelectionChangedEven
             KeySpaceComboBox.Items.Clear();
             TablesComboBox.Items.Clear();
             ColumnComboBox.Items.Clear();
+            string dbName = ((ComboBox)sender).SelectedItem.ToString();
+            db = (Database)EA.Dbs.First(m => m.Name == dbName);
+            if (db.DBType.Equals(eDBTypes.CosmosDb))
+            {
+                if (ValidationCfgComboBox.Items.Cast<ComboEnumItem>().Where(m => m.text.ToString().Equals("Insert")) == null
```

Comment:
use enum to compare the combo item eDBValidationType.Insert instead of hard coded string